### PR TITLE
Harimedha

### PR DESCRIPTION
**Pull request by medha-harimash**
sir please check me

### CODE REVIEW

File: public/js/custom.js [hunk 1]
```diff
@@ -306,12 +306,12 @@ var Samar = function(){
...
-					$('.site-header .container > .logo-header .logo').attr('src','images/logo.png');
-					$('.site-header .container > .logo-header .logo-2').attr('src','images/logo-2.png');
-					$('.site-header .container > .logo-header .logo-3').attr('src','images/logo-3.png');
+					$('.site-header .container > .logo-header .logo').attr('src','images/medha.png');
+					$('.site-header .container > .logo-header .logo-2').attr('src','images/medha.png');
+					$('.site-header .container > .logo-header .logo-3').attr('src','images/medha.png');
 				} else {
 					menu.removeClass('is-fixed');
-					$('.site-header .container > .logo-header .logo, .site-header .container > .logo-header .logo-2, .site-header .container > .logo-header .logo-3').attr('src','images/logo-white.png')
+					$('.site-header .container > .logo-header .logo, .site-header .container > .logo-header .logo-2, .site-header .container > .logo-header .logo-3').attr('src','images/medha.png')
```
Comment (medhaadmin) [inline public/js/custom.js]: this should be white logo to appear on color theme

Comment (medhaadmin) [reply public/js/custom.js]: please use this image: https://drive.google.com/file/d/1xnmXpwdm_xl9j3erEY7zT0uWDuLq1HIw/view?usp=sharing

File: resources/views/master.blade.php [hunk 2]
```diff
@@ -37,7 +37,7 @@
                 <div class="container clearfix">
                     <!-- Website Logo -->
                     <div class="logo-header">
-                        <a href="/"><img class="logo-3" src="images/logo-white.png" alt=""></a>
+                        <a href="/"><img class="logo-3" src="images/medha.png" alt=""></a>
```
Comment (medhaadmin) [inline resources/views/master.blade.php]: white logo as above

File: resources/views/master.blade.php [hunk 3]
```diff
@@ -102,9 +102,9 @@
                     <div class="col-xl-3 col-lg-12 col-md-4 col-sm-6 wow fadeIn" data-wow-duration="2s" data-wow-delay="0.2s">
                         <div class="widget widget_about">
                             <div class="footer-logo">
-                                <a href="index.html"><img src="images/logo-white.png" alt=""/></a>
+                                <a href="index.html"><img src="images/medha.png" alt=""/></a>
```
Comment (medhaadmin) [inline resources/views/master.blade.php]: white logo same as above

File: public/js/custom.js (hunk 1 above)
Comment (medha-harimash) [reply public/js/custom.js]: done

Comment (medha-harimash) [reply public/js/custom.js]: i have again done it

**Review by medhaadmin — COMMENTED** [review]
white logo: https://drive.google.com/file/d/1ZS77mNE-Vbyj1stseF6HCz4-lkkyIXjK/view?usp=sharing
color logo: https://drive.google.com/file/d/15l77pYpvYbbDnMyEawP8RmrU8hdzCRpq/view?usp=sharing

And please delete all unused files: public/favicon1.ico, public/favicon123.ico.png, public/images/TESTfavicon.png, public/images/favicon1.png, public/images/faviconhari.jpeg, public/images/medha_white1.png

**Review by medhaadmin — APPROVED** [review]
all good

**Pull request merged**